Remove the default resource group name.

dcaro opened the pull request:
This change to remove the default resource group name and inform that the name is also used for the fqdn of the vmss.

LuisCabrer commented:
#sign-off

azurecla commented:
Hi __@LuisCabrer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

dcaro committed:
Remove the default resource group name.

This change to remove the default resource group name and inform that the name is also used for the fqdn of the vmss.
